libjiffy/bench/fifo_queue_block_benchmark.cpp

```diff
@@ -46,20 +46,13 @@ int main(int argc, char const *argv[])
     }
     
     else if (vm.count("dram")) {
-        struct memkind* pmem_kind = nullptr;
         std::string memory_mode = "DRAM";
     }
-    if (vm.count("pmem")) {
-        struct memkind* pmem_kind = nullptr;
+    else if (vm.count("pmem")) {
         std::string memory_mode = "PMEM";
         std::string pmem_path = vm["pmem"].as<std::string>();
-        size_t err = memkind_create_pmem(pmem_path.c_str(),0,&pmem_kind);
-        if (err) {
-            char error_message[MEMKIND_ERROR_MESSAGE_SIZE];
-            memkind_error_message(err, error_message, MEMKIND_ERROR_MESSAGE_SIZE);
-            fprintf(stderr, "%s\n", error_message);
-        }
     }
+    void* pmem_kind = mem_utils::init_kind(memory_mode, pmem_path);
 
```

Comment:
This doesn't look like it will compile... memory_mode and pmem_path are defined in another scope.

Reply:
I did not build bench, my bad.